Python/Product/Debugger/DkmDebugger/PyObjectNativeVisualizer.cs

```diff
@@ -73,6 +73,7 @@ internal class PyObjectNativeVisualizer : DkmDataItem {
                 return null;
             }
 
+            // TODO: Localization ?
             var pyEvalResult = new PythonEvaluationResult(objRef, "[Python view]")
```

Comment:
I think so. Leave it for now and we'll see whether the loc testers call it out (though there's a real good chance they'll never see it - this is fairly advanced stuff).